complete/src/main/java/com/example/springboot/Application.java

```diff
@@ -21,12 +21,13 @@ public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
 
 			System.out.println("Let's inspect the beans provided by Spring Boot:");
 
-			String[] beanNames = ctx.getBeanDefinitionNames();
+			String beanNames = ctx.getBeanDefinitionNames();
```

Comment:
Removed unneeded brackets after String. 
Source: https://martinfowler.com/articles/microservices.html

Reply:
Are you sure that's not needed?  If getBeanDefinitionNames returns an array of Strings, it is needed.